src/plugins/dashboard/public/application/dashboard_app_controller.tsx

```diff
@@ -977,7 +977,7 @@ export class DashboardAppController {
             [UrlParams.SHOW_FILTER_BAR, 'filterBar', 'Filter bar'],
           ].map(([urlParam, translationId, defaultMessage]) => ({
             id: urlParam,
-            label: i18n.translate(`share.urlPanel.${translationId}`, {
+            label: i18n.translate(`dashboard.embedUrlParamExtension.${translationId}`, {
```

Comment:
Missed that earlier - to make it possible to statically collect all existing i18n ids, `translate` is only allowed to be called with a constant. It's a bit redundant sometimes but has a lot of benefits for the downstream process.

In that case I suggest:
```ts
const checkboxes = [
            [UrlParams.SHOW_TOP_MENU, i18n.translate('dashboard.embedUrlParamExtension.topMenu', { defaultMessage: 'Top menu' })],
            [UrlParams.SHOW_QUERY_INPUT, i18n.translate('dashboard.embedUrlParamExtension.query', { defaultMessage: 'Query'})],
            [UrlParams.SHOW_TIME_FILTER, i18n.translate('dashboard.embedUrlParamExtension.timeFilter', { defaultMessage: 'Time filter'})],
            [UrlParams.SHOW_FILTER_BAR, i18n.translate('dashboard.embedUrlParamExtension.filterBar', { defaultMessage: 'Filter bar'})],
].map(([urlParam, label]) => ({
            id: urlParam,
            label
```

Reply:
I've gone with a simple array of objects (without the map). Once Prettier had formatted it they took up the same number of lines, so this seems clearer even if it means the prop names are repeated.